pkg/grpc/authn/context_test.go

```diff
@@ -0,0 +1,25 @@
+package authn
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stackrox/rox/pkg/grpc/authn/mocks"
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/mock/gomock"
+)
+
+func TestCopyContextIdentity(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	mid := mocks.NewMockIdentity(ctrl)
+	mid.EXPECT().UID().AnyTimes().Return("username")
+
+	original := ContextWithIdentity(context.Background(), mid, t)
+	copy := CopyContextIdentity(context.Background(), original)
```

Comment:
nit: an additional entry could be added in original, but not set in copy in order to make sure that the original and copied contexts are different.